mne/time_frequency/tests/test_multitaper.py

```diff
@@ -3,7 +3,7 @@
 from numpy.testing import assert_array_almost_equal
 from distutils.version import LooseVersion
 
-from mne.time_frequency import dpss_windows, multitaper_psd
+from mne.time_frequency.multitaper import dpss_windows, _psd_multitaper
```

Comment:
It might actually be a better idea to test `psd_multitaper` so that you fully test the user facing function. So, if anything fails between the start of this function and the call to the private method, it will get caught ...